notify-github.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# MOVE THIS FILE TO  https://github.com/libero/scripts/ once tested
+set -e
+
+unique_id=$(uuidgen)
+temp_file="github-commit-status-${unique_id}.log"
+
+repository="libero/reviewer"
```

Comment:
also specific to this project, to be passed in?

Reply:
the code i tried would only give '/' so hardcoded it for now

Reply:
passed in as param